moulinette/actionsmap.py

```diff
@@ -368,17 +368,23 @@ def __init__(self, parser_class, namespaces=[], use_cache=True,
         for n in namespaces:
             logger.debug("loading actions map namespace '%s'", n)
 
-            if use_cache:
+            actionsmap_yml = '%s/actionsmap/%s.yml' % (pkg.datadir, n)
+            actionsmap_pkl = '%s/actionsmap/%s-%s.pkl' % (pkg.cachedir, n, os.stat(actionsmap_yml).st_size)
```

Comment:
It's not cryptographically safe, but it's really safe for our situation (I don't see this being used outside of development) and will lead to less false positives than modification time since it really cached up when the file is effectively modified in 99.9% situations.

If we want we could check both like rsync but I don't see that much additional value added by doing so.